### PR TITLE
[10.0][ADD] product_brand_pos_report

### PR DESCRIPTION
**Pull request by MPoch-PlanetaTIC**
Show product Brand in pos order report

cc @PlanetaTIC

### CODE REVIEW

File: product_brand_pos_report/reports/pos_order_report.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    config_id = fields.Many2one('pos.config', string='Point of Sale',
+                                readonly=True)
+    product_brand_id = fields.Many2one(
+        'product.brand',
+        string='Brand',
+    )
+
+    @api.model_cr
+    def init(self):
```
Comment (chienandalu) [inline product_brand_pos_report/reports/pos_order_report.py:20]: Well, this is monkeypatching the whole method. I guess in this version there's no better way to do it. Anyway, you should comment it and state possible future options.

Do this remark in the readme known issues section as well, as it won't be compatible with other modules trying to extend this report.

**Review by chienandalu — CHANGES_REQUESTED** [review]
Review linters: https://travis-ci.org/github/OCA/pos/jobs/678110483#L579-L582

File: product_brand_pos_report/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Brand POS Report',
+    'summary': 'Product Brand in Point of Sale Report',
+    'version': '10.0.1.0.0',
+    'description': 'Show product Brand in pos order report',
+    'category': 'Point Of Sale',
+    'author': 'PlanetaTIC',
+    'website': 'https://www.planetatic.com',
```
Comment (chienandalu) [inline product_brand_pos_report/__manifest__.py]:
```suggestion
    'website': 'https://www.github.com/OCA/pos',
```

File: product_brand_pos_report/reports/pos_order_report.py (hunk 1 above)
Comment (MPoch-PlanetaTIC) [reply product_brand_pos_report/reports/pos_order_report.py:20]: Fixed

File: product_brand_pos_report/__manifest__.py (hunk 2 above)
Comment (MPoch-PlanetaTIC) [reply product_brand_pos_report/__manifest__.py]: Fixed, thanks for the feedback

**Comment by legalsylvain** [comment]
@chienandalu : is it now ok for you ?

**Review by legalsylvain — APPROVED** [review]
LGTM. Code review, no test.

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**